### DISCUSSION TITLE
Problem occurs with <collection>Connection  when get values object.

### DISCUSSION THREAD

**Discussion by quangtuyennguyen** [strapi/strapi#8597] [Legacy Topics]
**Hello, everyone!**
Yesterday, this query worked fine but today did not.
I don't know how to fix this issue
Can someone help me?
Thanks all.
![2020-11-09_141616](https://user-images.githubusercontent.com/52388335/98511470-49e3bd00-2297-11eb-97c1-22f53253d9cf.png)

**Comment by derrickmehaffy** [comment]
Hello,

I see you are wanting to ask a question, we are currently planning to decomission our GitHub Discussions in early 2021 and we don't want to lose the great questions asked by our community.

Questions should be directed to [our forum](https://forum.strapi.io/). Please see the following contributing guidelines for asking a question [here](https://github.com/strapi/strapi/blob/master/CONTRIBUTING.md#reporting-an-issue).
